db/db_impl/db_impl_write.cc

```diff
@@ -1533,10 +1528,11 @@ Status DBImpl::HandleWriteBufferFull(WriteContext* write_context) {
 }
 
 uint64_t DBImpl::GetMaxTotalWalSize() const {
-  mutex_.AssertHeld();
-  return mutable_db_options_.max_total_wal_size == 0
-             ? 4 * max_total_in_memory_state_
-             : mutable_db_options_.max_total_wal_size;
+  auto max_total_wal_size = max_total_wal_size_.load(std::memory_order_acquire);
+  if (max_total_wal_size > 0) {
+    return max_total_wal_size;
+  }
+  return 4 * max_total_in_memory_state_;
```

Comment:
`max_total_in_memory_state_` is an atomic now, need to specify memory order as well?

Reply:
done